layouts/default.vue

```diff
@@ -59,14 +59,14 @@
         @click.stop="drawer = !drawer"
       />
       <v-btn
-        v-if="$route.name !== 'index'"
         :icon="opaqueAppBar || $vuetify.breakpoint.xsOnly || isScrolled"
         :fab="!(opaqueAppBar || $vuetify.breakpoint.xsOnly) && !isScrolled"
         :small="!(opaqueAppBar || $vuetify.breakpoint.xsOnly) && !isScrolled"
         :class="{
           'ml-n1': opaqueAppBar || $vuetify.breakpoint.xsOnly || isScrolled,
           'mr-2': !(opaqueAppBar || $vuetify.breakpoint.xsOnly) && !isScrolled,
-          'mr-1': opaqueAppBar || $vuetify.breakpoint.xsOnly || isScrolled
+          'mr-1': opaqueAppBar || $vuetify.breakpoint.xsOnly || isScrolled,
+          hidden: $route.name === 'index'
```

Comment:
Just to know, why the switch to display and not v-if?

Also maybe we could use `.d-none` from Vuetify classes instead of the custom one?

Reply:
Or we could keep the button, and on press redirect to the first page prior to using settings

Reply:
@Camc314 @ThibaultNocchi See #633

display: none (d-none) keeps the element in the DOM but it's not taken in account for layout calculations. Thus, it doesn't solve the problem.

Its mainly used for DOM manipulations in batches (so you can still do querySelection operations and such in them). See my recycle DOM PR: I append all the DOM elements first to the parent and, once they're all appended, use v-show (which is Vue internal of display: none) to show them, doing only one layout calculation (instead of one per dom element appended)

This means we need to keep the element in account for the layout calculations, so d-none or v-if (destroying the DOM node) doesn't work. We have two another options: ``opacity`` or ``visibility``. ``opacity`` effectively hides the element. However, it still reacts as if it was there, so you can click it and such. ``visibility``, in the other hand, it's exactly like ``display: none``, with the difference that it's taken in account for layout calculations.

Hope this explanation is helpful to understand this approach.